apps/condo/domains/news/tasks/notifyResidentsAboutNewsItem.spec.js

```diff
@@ -143,6 +145,98 @@ describe('notifyResidentsAboutNewsItem', () => {
+                }
+            )
+
+            // create a many newsItemScopes without Residents 
+            for (let i = 0; i <= 100; i++) {
+                await createTestNewsItemScope(adminClient, newsItem1, {
+                    property: { connect: { id: property.id } },
+                    unitType: FLAT_UNIT_TYPE,
+                    unitName: faker.lorem.word(),
+                })
+            }
+
+            // create last newsItemScope with created Resident
+            await createTestNewsItemScope(adminClient, newsItem1, {
+                property: { connect: { id: property.id } },
+                unitType: unitType1,
+                unitName: unitName1,
+            })
+
+            const payload = getRandomTokenData({
+                devicePlatform: DEVICE_PLATFORM_ANDROID,
+                appId: APP_RESIDENT_ID_ANDROID,
+                pushToken: getRandomFakeSuccessToken(),
+            })
+
+            await syncRemoteClientByTestClient(residentClient1, payload)
+
+            const messageWhere = { user: { id: residentClient1.user.id }, type: NEWS_ITEM_COMMON_MESSAGE_TYPE }
+
+            // Publish news item to make it send-able
+            const [updatedItem1] = await publishTestNewsItem(adminClient, newsItem1.id)
+            await notifyResidentsAboutNewsItem(newsItem1.id)
+
+            await waitFor(async () => {
+                const messages = await Message.getAll(adminClient, messageWhere)
+
+                expect(messages).toBeDefined()
+                expect(messages).toHaveLength(1)
+
+                const message1 = messages[0]
+
+                expect(message1).toBeDefined()
+                expect(message1.id).toMatch(UUID_RE)
+
+                expect(message1).toEqual(expect.objectContaining({
+                    status: MESSAGE_SENT_STATUS,
+                    processingMeta: expect.objectContaining({
+                        // old way check
+                        transport: 'push',
+
+                        // ADR-7 way check
+                        transportsMeta: [expect.objectContaining({
+                            transport: 'push',
+                        })],
+                    }),
+                    meta: expect.objectContaining({
+                        title: 'Lorem ipsum dolor sit amet, consectetur...',
+                        body: 'Commodo viverra maecenas accumsan lacus vel facilisis volutpat est velit. Et malesuada fames ac turpis egestas sed tempus urna et. At augue eget...',
```

Comment:
Why body differs from payload above? If you test crop logic, test it programmatically, not hard-coded